From 67973ffa137b8d85248046687fe3c2b50d43c210 Mon Sep 17 00:00:00 2001
From: Lionel Xu <Lionel.Xu@freescale.com>
Date: Wed, 27 Jul 2011 17:23:58 +0800
Subject: [PATCH 0303/2508] ENGR00153793-2 ESAI: add function get_rate to clk
 pll3_pfd_508M

Add function get_rate to clk pll3_pfd_508M.

Signed-off-by: Lionel Xu <R63889@freescale.com>
---
 arch/arm/mach-mx6/clock.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx6/clock.c b/arch/arm/mach-mx6/clock.c
index a76c57f..edac9b8 100644
--- a/arch/arm/mach-mx6/clock.c
+++ b/arch/arm/mach-mx6/clock.c
@@ -613,6 +613,7 @@ static struct clk pll3_pfd_508M = {
 	.enable = _clk_pfd_enable,
 	.disable = _clk_pfd_disable,
 	.set_rate = pfd_set_rate,
+	.get_rate = pfd_get_rate,
 	.round_rate = pfd_round_rate,
 };
 
-- 
1.7.7.4

